libs/portal-framework-ui/src/components/data-table/ToolbarRenderer.tsx

```
@@ -32,12 +34,20 @@ function ActionItemRenderer<TData extends BaseRecord>(
   const actionItem = getAction<TData>(item.id);
   if (!actionItem) return null;
 
+  // Check if we're on mobile to adjust button sizing
+  const { isMobile } = useMobileDetection({
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)